pages/getData.tsx

```diff
@@ -210,7 +244,11 @@ const GetData: NextPage = () => {
                 </div>
               </article>
               <pre style={{ wordBreak: 'break-all', whiteSpace: 'pre-wrap' }}>
-                {data}
+                {dataKey.substring(0, 26) == '0x4b80742de2bf82acb3630000'
```

Comment:
I suggest to add context and to give a name to this: `0x4b80742de2bf82acb3630000` - because we we see this we have no idea what this value is. Add this in a variable so we can give it a name:

```
const ADDDRESS_PERMISSIONS_PERMISSION_PREFIX = '0x4b80742de2bf82acb3630000';
```

or something